### PR TITLE
Fixed docstrings, fixed parsing

### PR DESCRIPTION
**Pull request by ordirules**
proposed fix for #80. If this is good, maybe we should also add a test by testing visualize on the lambda function.

### CODE REVIEW

**Comment by ordirules** [comment]
related to #76

**Comment by mrocklin** [comment]
Testing would be good if you're up for it. We might also consider
whitelisting rather than blacklisting.  For example we might only pass
through characters that are alphanumeric.

**Comment by ordirules** [comment]
Actually, all that was needed in the test was @CJ-Wright 's suggestion in #76 so I will cherry pick it. Thanks @CJ-Wright.

I also added the white list and a test for the string parser.

**Comment by ordirules** [comment]
Also @CJ-Wright said in #76:

> Also it seems that we could use a more robust assert than just checking the file's existence.

Any suggestions?

File: streamz/tests/test_graph.py [hunk 1]
```diff
@@ -27,7 +29,7 @@ def test_create_file():
     source2 = Stream(stream_name='source2')
 
     n1 = source1.zip(source2)
-    n2 = n1.map(add).scan(mul)
+    n2 = n1.map(add).scan(mul).map(lambda x : x + 1)
```
Comment (ordirules) [inline streamz/tests/test_graph.py]: @CJ-Wright 's suggestion

**Comment by mrocklin** [comment]
Great.  Merging.  Thanks @ordirules !

**Pull request merged**